src/plugins/wb-data-json/data-json.js

```diff
@@ -325,10 +325,10 @@ var componentName = "wb-data-json",
 					}
 
 					// Set the value to the node
-					if ( $.isArray( cached_value ) ) {
-						applyTemplate( cached_node, j_cache, cached_value );
-					} else if ( j_cache.isHTML ) {
+					if ( j_cache.isHTML ) {
 						cached_node.innerHTML = cached_value;
+					} else if ( $.isArray( cached_value ) || cached_value && !( cached_value instanceof String ) && typeof cached_value === "object" ) {
```

Comment:
Good job avec cette conditionnelle, love it! This part requires deep knowledge of Javascript for sure:
`!( cached_value instanceof String ) && typeof cached_value === "object"`